sw/device/silicon_creator/rom/e2e/boot_policy_newer/BUILD

```diff
@@ -121,42 +102,35 @@ BOOT_POLICY_NEWER_CASES = [
     },
 ]
 
-[otp_image(
-    name = "otp_img_boot_policy_newer_{}".format(lc_state),
-    src = "//hw/ip/otp_ctrl/data:otp_json_{}".format(lc_state),
-    overlays = STD_OTP_OVERLAYS,
-) for lc_state, _ in get_lc_items()]
-
-# Splice OTP images into bitstreams
 [
-    bitstream_splice(
-        name = "bitstream_boot_policy_newer_{}".format(lc_state),
-        src = "//hw/bitstream:rom_with_fake_keys",
-        data = ":otp_img_boot_policy_newer_{}".format(lc_state),
-        meminfo = "//hw/bitstream:otp_mmi",
-        update_usr_access = True,
+    otp_image(
+        name = "otp_img_boot_policy_newer_{}".format(lc_state),
+        src = "//hw/ip/otp_ctrl/data:otp_json_{}".format(lc_state),
+        overlays = STD_OTP_OVERLAYS,
     )
     for lc_state, _ in get_lc_items()
 ]
 
 [
-    opentitan_functest(
+    opentitan_test(
         name = "boot_policy_newer_{}_a_{}_b_{}".format(
             lc_state,
             t["a"],
             t["b"],
         ),
         cw310 = cw310_params(
-            bitstream = ":bitstream_boot_policy_newer_{}".format(lc_state),
+            assemble = "{fw_a}@0 {fw_b}@0x80000",
```

Comment:
maybe could use `SLOTS` in `//sw/device/silicon_creator/rom/e2e:defs.bzl`

Reply:
Done.